docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -1368,6 +1327,22 @@ This section contains reference information about each of the configuration prop
 |`fhirServer/audit/serviceProperties/geoState`|string|The Geo State configure for CADF audit logging service.|
 |`fhirServer/audit/serviceProperties/geoCounty`|string|The Geo Country configure for CADF audit logging service.|
 |`fhirServer/search/useBoundingRadius`|boolean|True, the bounding area is a Radius, else the bounding area is a box.|
+|`fhirServer/bulkdata/applicationName`| string|Fixed value, always set to fhir-bulkimportexport-webapp |
+|`fhirServer/bulkdata/moduleName`|string| Fixed value, always set to fhir-bulkimportexport.war |
+|`fhirServer/bulkdata/jobParameters/cos.bucket.name`|string|Object store bucket name |
+|`fhirServer/bulkdata/jobParameters/cos.location`|Object store location |
+|`fhirServer/bulkdata/jobParameters/cos.endpointurl`| Object store end point url |
+|`fhirServer/bulkdata/jobParameters/credential.ibm`| If use IBM credential, "Y" or "N" |
+|`fhirServer/bulkdata/jobParameters/cos.api.key`|string|API key for accessing IBM COS |
+|`fhirServer/bulkdata/jobParameters/cos.srvinst.id`|string|Service instance Id for accessing IBM COS |
+|`fhirServer/bulkdata/implementation_type`|string|"cos" or "dummy" which matches the desired bulk operation implementation type |
```

Comment:
does "cos" cover all S3-compatible object stores or just the IBM Cloud one?

Reply:
I think the answer is all, let me change the description a little bit. 
IBM COS client SDK is forked from Amazon S3 SDK.

Reply:
done

Reply:
I think we should say this in the "description" column of this row. Something like this:
> Use "cos" for any S3-compatible object store

Reply:
en, interesting, seems my change doesn't show up here. I changed to "  "cos" (Any Amazon S3-compatible object stores including IBM COS) or "dummy" which matches the desired bulk operation implementation type ".     but I do think your description is better, let me change to it.

Reply:
done